server/events/command_context.go

```diff
@@ -37,4 +37,10 @@ type CommandContext struct {
 	// set our own build statuses which can affect mergeability if users have
 	// required the Atlantis status to be successful prior to merging.
 	PullMergeable bool
+	// AzureDevopsWorkItemID Azure Devops Atlantis commands come from a work item
+	// comment with a link to one or more pull requests. Pull requests may
+	// also be linked to more than one work item, so we need to know which work item
+	// triggered this pull request command.
+	// https://docs.microsoft.com/en-us/azure/devops/boards/queries/link-work-items-support-traceability?view=azure-devops&tabs=new-web-form
+	AzureDevopsWorkItemID int
```

Comment:
Is this used?

Reply:
It doesn't appear to be, good catch, thanks.  In the azuredevops_client CreateComment() function, I decided to log an error and continue if a user has a pull request linked to multiple work items.  I hope that such a thing would be an edge case, if anything.  I can remove that struct member and any associated comments/example code that is left behind.